flow/flow.cpp

```diff
@@ -21,9 +21,24 @@
 #include "flow/flow.h"
 #include "flow/DeterministicRandom.h"
 #include "flow/UnitTest.h"
+#include "flow/rte_memcpy.h"
+#include "flow/folly_memcpy.h"
 #include <stdarg.h>
 #include <cinttypes>
 
+#if defined (__linux__) || defined (__FreeBSD__)
+// For benchmarking; need a version of rte_memcpy that doesn't live in the same compilation unit as the test.
+void * rte_memcpy_noinline(void *__restrict __dest, const void *__restrict __src, size_t __n) {
```

Comment:
It can't be in the same compilation unit as the benchmark, so it's either here (next to the other  memcpy stuff), or in a special .cpp all by itself.  The latter seems like overkill.